server/fishtest/templates/tests_view.mak

```diff
@@ -38,7 +41,7 @@
     </script>
 % endif
 
-<div id="enclosure" ${"style=\"visibility:hidden !important;\"" if show_task >= 0 else "" |n}>
+<div id="enclosure" ${"style=\"visibility:hidden;\"" if show_task >= 0 else "" |n}>
```

Comment:
For consistency, I think it is better to use a combination of single quotes and double quotes,
something like:
`<div id="enclosure" ${'style="visibility:hidden;"' if show_task >= 0 else "" |n}>`
would be better read no?